README.md

```diff
@@ -1,2 +1,17 @@
-# echo-heaps
-simple example for integrating [echo physics](https://austineast.dev/echo/) with the [heaps engine](https://heaps.io)
+# What is this
+
+- This is a game that integrating [echo physics](https://austineast.dev/echo/) with the [heaps engine](https://heaps.io)
+
+## How to start
+```
```

Comment:
Could you change this section to use this text instead?

Clone the repo, then install the project's dependencies:
`haxelib install all `

To produce hashlink target:
`haxe hl.hxml`

To produce javascript target:
`haxe js.hxml`